test/fixtures/indices.yml

```diff
@@ -0,0 +1,15 @@
+# Read about fixtures at http://api.rubyonrails.org/classes/ActiveRecord/FixtureSet.html
+
+one:
+  finalized_at: 2016-01-13 16:11:26
+  category_id: :one
```

Comment:
If you create the resource using  `category:references`,  it would generate the fixtures like 

```
one:
  category: 
```

When you say  `category_id:integer`  you get 
```
one:
  category_id: 1
```

Which you would then have to fix by hand.  You should probably always be using `model:references` when creating migrations.